.travis.yml

```diff
@@ -26,7 +32,8 @@ install:
   - composer --verbose install
 
 script:
-  - if [[ $RELEASE = dev ]]; then composer --verbose require --no-update drupal/core:8.4.x-dev; fi;
+  - if [[ $RELEASE = dev ]]; then composer --verbose remove drupal/console; fi;
```

Comment:
The documentation says it should...?

https://getcomposer.org/doc/03-cli.md#remove

--no-update should remove drupal/console from the composer.json without triggering an update which happens in the next line.